adyenv6core/resources/localization/adyenv6core-locales_en.properties

```diff
@@ -60,6 +60,9 @@ type.paymentinfo.adyenSocialSecurityNumber.name=Social Security Number
 type.paymentinfo.adyenFirstName.name=First Name
 type.paymentinfo.adyenLastName.name=Last Name
 type.paymentinfo.adyenBoletoUrl.name=Boleto URL
+type.paymentinfo.adyenBoletoExpirationDate.name=Boleto expiration date
```

Comment:
Can you please make first letter capital for each word as we are using for other labels.